object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,22 @@
+/* exported toObject */
+/* Make a holder for the array.This will be to contain temporary values.
+  -Look at each number from 0, check if the current number is less than the length of the input, and
+  increment it by 1.
+  -For each number:
+    -if the remainder of the current number divided by 2 is strictly equal to 0:
+      -create a holder for the key value pairs.
+      -The input value at the position of the current number plus 1, is assigned to
+      the holder created previously at the input at the current number.
+      -Add the previous value to the holder for the key value pairs
+      -Return the object */
+function toObject(keyValuePair) {
+  var arr = [];
+  for (var i = 0; i < keyValuePair.length; i++) {
+    if (i % 2 === 0) {
+      var x = {};
+      x[keyValuePair[i]] = keyValuePair[i + 1];
+      arr.push(x);
+    }
+  }
```

Comment:
`keyValuePair` will only ever have two indexes, no loop is necessary here

Reply:
<img width="720" alt="Screen Shot 2022-07-16 at 2 51 28 PM" src="https://user-images.githubusercontent.com/97363006/179372974-4281000a-9277-44c2-b9d9-c43b49569cb4.png">